x/participationrewards/keeper/callbacks_test.go

```diff
@@ -82,17 +82,21 @@ func (suite *KeeperTestSuite) executeValidatorSelectionRewardsCallback(performan
 	rewardsQuery := distrtypes.QueryDelegationTotalRewardsRequest{DelegatorAddress: performanceAddress}
 	bz := prk.GetCodec().MustMarshal(&rewardsQuery)
 
-	qid := icqkeeper.GenerateQueryHash(
+	// query, found := prk.IcqKeeper.GetQuery(ctx, qid)
+	// s.Require().True(found, "qid: %s", qid)
+
+	// Should we create a new query first since the query with the qid generate above i dont see where it was set in the store
```

Comment:
I believe it is created during the KeeperTestSuite setup when the epoch passes (and never gets responded to).